tests/js/spec/views/groupDetails/groupEventDetails.spec.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import {mount} from 'enzyme';
+
+import {initializeOrg} from 'app-test/helpers/initializeOrg';
+import GroupEventDetails from 'app/views/groupDetails/shared/groupEventDetails';
+
+describe('groupEventDetails', () => {
+  let org;
+  let event;
+  let group;
+  let project;
+
+  beforeEach(() => {
+    org = TestStubs.Organization();
+    group = TestStubs.Group();
+    project = TestStubs.Project({organization: org});
+
+    event = TestStubs.Event({
+      size: 1,
+      dateCreated: '2019-03-20T00:00:00.000Z',
+      errors: [],
+      entries: [],
+    });
+
+    MockApiClient.addMockResponse({
+      url: `/issues/${group.id}/`,
+      body: group,
+    });
+
+    MockApiClient.addMockResponse({
+      url: `/issues/${group.id}/events/latest/`,
+      body: event,
+    });
+
+    MockApiClient.addMockResponse({
+      url: `/projects/${org.slug}/${project.slug}/issues/`,
+      method: 'PUT',
+    });
+
+    MockApiClient.addMockResponse({
+      url: `/projects/${org.slug}/${project.slug}/events/${event.id}/committers/`,
+      body: {committers: []},
+    });
+
+    MockApiClient.addMockResponse({
+      url: `/projects/${org.slug}/${project.slug}/events/${event.id}/owners/`,
+      body: {owners: [], rules: []},
+    });
+
+    MockApiClient.addMockResponse({
+      url: `/issues/${group.id}/participants/`,
+      body: [],
+    });
+
+    MockApiClient.addMockResponse({
+      url: `/issues/${group.id}/tags/`,
+      body: [],
+    });
+
+    MockApiClient.addMockResponse({
+      url: `/groups/${group.id}/integrations/`,
+      body: [],
+    });
+
+    MockApiClient.addMockResponse({
+      url: '/sentry-apps/',
+      body: [],
+    });
+
+    MockApiClient.addMockResponse({
+      url: `/organizations/${org.slug}/sentry-app-installations/`,
+      body: [],
+    });
```

Comment:
There we no tests for the `groupEventDetails` component as far as I could tell. Seems a bit scary since I _think_ it's the top-level component for the whole Issues page?

Also, the above was rough – is there a better way of stubbing/setting up a world for the Issue page to be mounted and tested here or is all the above just what we have to do?

Reply:
Yeah that's a bit concerning but we do have percy snapshots on those pages though.

Reply:
A screenshot isn't a good replacement for testing _functionality_ or how things interact with one another, imo. 🤷‍♂️

Reply:
I don't disagree, I'm just suggesting why it's been neglected thus far

Reply:
Yea, I wasn't trying to imply you were defending it – sorry if it came off like that. I was just throwing out that I think we rely on Percy too much.